src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Model/UserAuthenticationResult.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using Altinn.Platform.Authentication.Enum;
+using Microsoft.AspNetCore.Authentication;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Authentication.Model
+{
+    /// <summary>
+    /// Handles UserProfile
+    /// </summary>
+    [DataContract]
```

Comment:
I suspect that all the attribute annotations in this class are unnecessary. We never serialize this model to or from XML and the code that deserialize this model from JSON is using the System.Text.Json namespace:
`JsonSerializer.Deserialize<UserAuthenticationResult>(content)`

Reply:
You are correct, they are now reduntant. I removed it from the attributes.